doc/developer_guide/psyir_symbols.rst

```diff
@@ -105,6 +105,9 @@ explicitly listed may be assumed to be unsupported):
 +----------------------+--------------------+--------------------+
 |                      |PUBLIC, PRIVATE     |                    |
 +----------------------+--------------------+--------------------+
+|                      |Data statements     |                    |
```

Comment:
Thanks for updating this. Since I couldn't remember what data statements were and had to google it, perhaps they should be included in the "initialisation expressions" cell above, e.g. "Initialisation expressions (incl. limited support for `data` statements)"

Reply:
OK, I thought I had added it to that section (though I now realised that below are other non-init statements). TBH, I found that table a bit confusion. For now I have tried to move the 'Initialisation Expressions' to the left column, and then data statements to middle column next to it. Is that how you intended this to be?

Reply:
Sort of. It was supposed to cover initialisation in variable declarations too although I agree it wasn't very clear. Perhaps we should add "explicit initialisation" (as that's what the standard calls it) to the "supported" cell too?